tests/cephadm/test_cephadm_ansible_wrapper.py

```diff
@@ -0,0 +1,72 @@
+"""Test module for ceph-ansible wrapper."""
+
+import os
+
+from ceph.ceph import ResourceNotFoundError
+from ceph.ceph_admin.cephadm_ansible import CephadmAnsible
+from utility.log import Log
+
+logger = Log(__name__)
+
+CEPHADM_ANSIBLE_PATH = "/usr/share/cephadm-ansible"
+
+
+def _get_node_ip(ceph_cluster, node_name):
```

Comment:
As per our discussion,
1. please modify _get_node_ip(ceph_cluster, node_name) module  such that **node_name** parameter takes dictionary 
2. _get_node_ip **could** be moved to ceph.py for re-usability purpose